CHANGELOG.md

```diff
@@ -33,14 +33,18 @@
   [SimplyDanny](https://github.com/SimplyDanny)
   [#5952](https://github.com/realm/SwiftLint/issues/5952)
 
+* Remove lints for `redundant_sendable` on protocols, where it is not redundant.
```

Comment:
```suggestion
* Remove lints for `redundant_sendable` on protocols, where `Sendable` is not redundant.  
```

Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.